src/jvm/test/org/jetbrains/kotlinx/lincheck/test/runner/ResumingFollowUpTest.kt

```diff
@@ -28,7 +28,7 @@ import org.jetbrains.kotlinx.lincheck.verifier.linearizability.LinearizabilityVe
 import org.junit.Test
 import kotlin.coroutines.*
 import kotlin.coroutines.intrinsics.*
-import org.jetbrains.kotlinx.lincheck.actor
+import org.jetbrains.kotlinx.lincheck.dsl.actor
```

Comment:
Please don't change the package

Reply:
I think it's convenient to store both Java and Kotlin DSLs in the one "dsl" package, isn't it?

Reply:
Changed CustomScenarioKotlinDSL.kt file's package and supressed 'PackageDirectoryMismatch'